Licensing_Program/userfiles_handling.py

```diff
@@ -7,11 +7,10 @@
 import re
 import tempfile
 
-import license_handling
-
-
-HEADER_SIGNAL_STRING = "Copyright"
 HEADER_IN_FIRST_N_LINES = 20
+HEADER_SIGNAL_STRING = "Copyright"
+
+import license_handling
 
```

Comment:
Darn. Okay, it's probably better to save removing this dependency for a different PR then.